src/lib/apis/convection.js

```diff
@@ -9,7 +10,7 @@ export default (path, accessToken, fetchOptions = {}) => {
   const headers = { Accept: "application/json" }
   if (accessToken) assign(headers, { Authorization: `Bearer ${accessToken}` })
   return fetch(
-    `${CONVECTION_API_BASE}/${path}`,
```

Comment:
This, as well as other instances, runs the risk of forming URLs like `http://domain//path`

Reply:
Interesting! I guess we've always just assumed they would be set properly, but I totally get what you mean.

I'm not super familiar with the _best_ way to do this (ie- `urljoin`, something built in perhaps?), but I love the idea 👍  

I ❤️ this